app/controllers/shelters_controller.rb

```diff
@@ -78,11 +78,24 @@ def drafts
   # This is the definition of a beautiful hack. 1 part gross, 2 parts simplicity. Does something neat not clever.
   def set_headers
     if(admin?)
-      @columns = Shelter::ColumnNames + Shelter::PrivateFields
-      @headers = Shelter::HeaderNames + Shelter::PrivateFields.map(&:titleize)
+      columns = Shelter::ColumnNames + Shelter::PrivateFields
+      @columns = columns
+      @headers = columns.map(&:titleize)
     else
       @columns = Shelter::ColumnNames
-      @headers = Shelter::HeaderNames
+      @headers = Shelter::ColumnNames.map(&:titleize)
```

Comment:
Is this a typo? Should it be HeaderNames?

Reply:
In this particular method, I'm not a huge fan of `HeaderNames`, as it's not explicit that it refers *only* to the headers of the set of default columns in `ColumnNames`. Given that we have references to three different sets of column names (`ColumnNames`, `PrivateFields`, and now `IndexHiddenColumnNames`), it seemed helpful to keep everything explicit here and just call `:titleize` on the raw column names right here.